src/software/backend/backend.h

```diff
@@ -17,10 +17,21 @@
 class Backend : public Subject<SensorMsg>,
                 public Subject<World>,
                 public Subject<RobotStatus>,
+                public ThreadedObserver<World>,
```

Comment:
I'm a bit confused on why this receives a `World` now?

Reply:
According to the [doc](https://docs.google.com/document/d/13BExBYrU6o96WMQGdhUzqdiBatNYfe-HLZJH3kaj3fo/edit?usp=sharing), `Backend` will consume `World` from `SensorFusion` to send vision packets to the robots.

Reply:
Ah, and eventually it will stop sending out `World` in favor of raw data right? I think my confusion was from having it consume and produce `World`.

Reply:
yes this is to support the old interface and new interface simultaneously